cluster/ubuntu/util.sh

```diff
@@ -447,7 +580,8 @@ function kube-down {
 # Perform common upgrade setup tasks
 function prepare-push() {
   # Use local binaries for kube-push
-  if [[ "${KUBE_VERSION}" == "" ]]; then
+  if [[ -z ${KUBE_VERSION} ]]; then
```

Comment:
You want to quote the variable reference here, in case (at least) it expands to the empty string.